universal/styles/variables.css

```diff
@@ -0,0 +1,8 @@
+/* colors */
+@value primary: #BF4040;
+@value secondary: #1F4F7F;
+
+/* breakpoints */
+@value small: (max-width: 599px);
+@value medium: (min-width: 600px) and (max-width: 959px);
+@value large: (min-width: 960px);
```

Comment:
sure, i'll actually just remove that for now. I just copied some code from the docs to get us started. thanks, great catch